### ISSUE TITLE
Update to latest version of Raven (3.5)

### ISSUE THREAD

**Issue by huard** [CSHS-CWRA/RavenPy#156]
### Description

There is a new release of Raven (3.5) with new algorithms for stream temperature calculations. 

- [ ] Build conda package
- [ ] Run tests with current RavenPy
- [ ] Adjust RavenPy as needed

**Comment by Zeitsperre** [comment]
Conda package for Raven v3.5: https://github.com/conda-forge/raven-hydro-feedstock/pull/5

Waiting on the go-ahead from @julemai before we move forward. In all likelihood, early-mid April.

**Comment by dingxinjun** [comment]
ravenpy can't installed on windows platform

**Comment by huard** [comment]
Yes, that's something we've been working on, but that has not yet been released. What Trevor did was make the Raven executable installable on Windows using conda. The next step will be to update RavenPy to take advantage of that, but your info, it's not high on our list of priorities.

**Comment by huard** [comment]
Raven 0.10.0 supports Raven 3.5
